kubespawner/reflector.py

```diff
@@ -218,9 +218,12 @@ async def _list_and_update(self):
         kwargs = dict(
             label_selector=self.label_selector,
             field_selector=self.field_selector,
+            resource_version=resource_version,
```

Comment:
Should we set this only if it's not `None`?

Reply:
As it is confusing what unset means in the api and its clients Ill update the code to use None and explicitly not add it to the kwargs